Software/app/freefall_lorawan/README.md

```diff
@@ -0,0 +1,76 @@
+# freefall\_lorawan
+
+## Description
+
+This application is used to detect free fall events and also serves as a template for acceleration detection handling. Each time a free fall event is detected by the accelerometer, a message containing a counter in the payload is sent. The transmitted value is incremented each time the event occurs. Additionally, if a downlink is received over the correct port, which can be configured by the user, the buzzer will beep periodically to notify locally that the GNSE should be picked up. The buzzer can be turned off by pressing the user button.
+
+## Configuration
+
+### Default
+1. Follow [this](https://www.thethingsindustries.com/docs/devices/adding-devices/) guide to add your device to TTN.
+2. Change `APPEUI`, `DEVEUI` and `APPKEY` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h) to the values for your specific device on TTN.
+3. Uncomment the REGION parameters in [conf/lorawan_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/lorawan_conf.h) and set `ACTIVE_REGION` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/lora_app.h). These should correspond to the location that you use these devices for.
+
+### Free fall detection
+1. Change `ACC_FF_THRESHOLD` in [app/conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h) to your liking. This will determine the threshold where all axis can be from 0g (which would indicate that it is free falling) to be considered a free fall event.
+2. Change `ACC_FF_DURATION` in [app/conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h) to your liking. This will determine how many times the axis have to meet the threshold values. Note that `ACC_FF_ODR` in the same file also has influence on this.
```

Comment:
```suggestion
2. Change `ACC_FF_DURATION` in [app/conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/freefall_lorawan/conf/app_conf.h) to a value you prefer. This will determine how many times the axis have to meet the threshold values. Note that `ACC_FF_ODR` in the same file also affects this.
```